infra/ui-base/src/main/java/com/simprints/infra/uibase/camera/qrscan/CameraFocusManager.kt

```diff
@@ -1,4 +1,4 @@
-package com.simprints.feature.login.tools.camera
+package com.simprints.infra.uibase.camera.qrscan
```

Comment:
I am a bit conflicted about this move. On one hand, it is an easy way to reuse without micro-modules. On the other hand, ui-base is added to every feature module, and that means that all of them get camera and QR handling added for no real reason.

Would be interested to hear others' opinions on this.